apps/aecore/src/aec_hard_forks.erl

```diff
@@ -70,6 +72,21 @@ ensure_env() ->
 protocol_effective_at_height(Height) ->
     protocol_effective_at_height(Height, protocols()).
 
+-spec protocol_vsn_name(protocol_vsn()) -> atom().
```

Comment:
I see this is not being used but it is rather a complementary function to protocol_vsn/1

Reply:
This was used in (now removed) generic "assert_min_protocol" function usable in aeprimop, which was able to turn protocol version number to nicer `{required, iris, some_feature_name}` failure. 
But since the check was moved to fate level, it's not used right now... We should refactor the min protocol requirements in aeprimop (and tests) to use that to get nicer errors...